rho/postprocessing.py

```diff
@@ -813,21 +821,31 @@ def process_fuse_on_eap(fact_names, host_vars):
 def process_karaf_home(fact_names, host_vars):
     """Process karaf_home indicators to detect Fuse-on-Karaf."""
 
-    if JBOSS_FUSE_ON_KARAF_KARAF_HOME not in fact_names:
+    log.info('In process_karaf_home')
+
+    if (JBOSS_FUSE_ON_KARAF_KARAF_HOME not in fact_names and
+            JBOSS_FUSE_SUMMARY not in fact_names):  # noqa
         return {}
 
+    log.info('Past first return')
```

Comment:
Is this logging valuable?

Reply:
Sorry, I left a bunch of debug logging in. I took it all out.